src/OrbitGl/OrbitApp.cpp

```diff
@@ -2218,7 +2218,9 @@ const orbit_client_protos::TimerInfo* OrbitApp::selected_timer() const {
 }
 
 void OrbitApp::SelectTimer(const orbit_client_protos::TimerInfo* timer_info) {
-  if (timer_info != nullptr && !IsTimerActive(*timer_info)) return;
+  if (timer_info != nullptr && !IsTimerActive(*timer_info) &&
+      timer_info->type() != TimerInfo::kCoreActivity)
```

Comment:
Nit: I think this at least deserves a brief comment, why we can select an inactive core activity timer.